### ISSUE TITLE
Intellisense for output data

### ISSUE THREAD

**Issue by jwillmer** [elsa-workflows/elsa-core#1321]
As a user I would like to access the output data of the previous (another) node in my current node. I would like to have intellisense/autocomplete when accessing the (json?) data. Something like:

```
activities.arrival_data.Output().my_payload..;
```

Blocked by #671

**Comment by tomy2105** [comment]
Will this work also with older notation (without output properties) with just ActivityName.Output ?

**Comment by sfmskywalker** [comment]
No, but we could add it back for existing workflows to prevent them from breaking and somehow communicate that it's a deprecated syntax.

**Comment by tomy2105** [comment]
It would definitely be nice if "deprecated" Output also had intelisense.....

**Comment by sfmskywalker** [comment]
Closing as stale because this request is for the legacy Elsa 2 designer experience in `elsa-core`. Current designer work lives in `elsa-studio`, so I am not keeping old UI enhancement backlog items open here.
